playhouse/db_url.py

```diff
@@ -42,6 +51,11 @@ def parseresult_to_dict(parsed):
     elif 'sqlite' in parsed.scheme and not connect_kwargs['database']:
         connect_kwargs['database'] = ':memory:'
 
+    # Get additional connection args from the query string
+    connect_kwargs.update(
```

Comment:
Can you not just do `connect_kwargs.update(parse_qsl(query))`? I think dict.update takes a tuple-of-2-tuples.